src/App.tsx

```diff
@@ -15,36 +21,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHABETICALLY = 'alphabetically';
```

Comment:
Valid approach, but cleaner way to do it  will be to create a type for it, like:

```
enum SortType {
  ALPHABETICALLY = 'alphabetically',
  BY_LENGTH = 'length',
}
```

Or a simple object, in case you work with js without ts.